src/model/neuroevolution/individual.py

```diff
@@ -165,9 +161,7 @@ def multiply_shuffle_neuron_weights_mutation(individual: "ChildIndividual") -> N
         """
         Multiplies all weights of one random neuron by random numbers and shuffles them.
 
-        Multiplies all weights for a single randomly chosen neuron,
-        from randomly chosen layer, by random numbers.
-        After weights multiplication by random numbers, the weight are shuffled.
+        The individual's layer is chosen randomly.
```

Comment:
What layer? There is nothing that says anything about layers, so this is confusing.

Reply:
The same layer which is mentioned in other mutation functions. Same for `weights` and `neurons`.